blogs/8_berkeley_function_calling_leaderboard.html

```diff
@@ -702,6 +702,11 @@ <h5>Parameter Type & Value Matching</h5>
                                 <li style="margin-bottom: 5px;">For <code>String</code>:
                                     <ul>
                                         <li>The evaluation process is <i>case-insensitive</i>.</li>
+                                        <li>All strings will be standardized before checking against the possible answers (which are also standardized). </li>
+                                        <ul>
+                                            <li>All white space is removed.</li>
+                                            <li>A subset of punctuations <code>,./-_*^</code> are removed as well to make the evaluation more robust and accurate.</li>
+                                        </ul>
```

Comment:
Improve wordings here:

<li>All strings, including both model generated string parmater values and possible answers, will be standardized before checking against the possible answers. </li>
                                        <ul>
                                            <li>All white spaces is removed.</li>
                                            <li>A subset of punctuations <code>,./-_*^</code> are removed to make the evaluation more robust and accurate.</li>
                                        </ul>